### PR TITLE
Added documentation for the email notifications

### PR DESCRIPTION
**Pull request by rumiahkessel**
I added instructions for how a user can setup email and slack notifications for their jobs. 

Issue: https://github.com/centerforaisafety/cerberus-cluster/issues/47

### CODE REVIEW

**Review by trent-safe-ai — APPROVED** [review]
looks good. merge away!

File: _posts/2023-05-03-cais-cluster-documentation.md [hunk 1]
```diff
@@ -26,9 +26,10 @@ title: Welcome to the Center for AI Safety Cluster
     1. [Old GCC, G++, Glibc](#how-to-update-gcc,-g++,-or-glibc)
     1. [Example sbatch file](#example-sbatch-file)
     1. [Docker Support](#docker-support)
-1. Links to other pages
+    1. [Configuring Notifications](#configuring-notifciations)
+2. Links to other pages
```
Comment (steven-basart) [inline _posts/2023-05-03-cais-cluster-documentation.md:30]: Might be good to install a markdown linter. Markdown will automatically renumber all the "1"s into the correct number. Better than having a person renumber everything every time they want to make a change.

File: _posts/2023-05-03-cais-cluster-documentation.md [hunk 2]
```diff
@@ -300,6 +301,21 @@ gcc --version # if you print it out again here it'll be version 10
 sleep 5
 ```
 
+## Configuring Notifications
+
+We have configured the [goslmailer](https://github.com/CLIP-HPC/goslmailer) on the cluster to provide job status notifications. This mailer is capable of sending email and slack notifications for various job stages (begin, fail, requeue, complete, etc). It is also capable of interfacing with other notification platforms, so if you would like us to configure it for another platform let us know and we will look into it.
+
+You can add notifications to your job by adding the following lines to your SBATCH file.
+```bash
```
Comment (steven-basart) [inline _posts/2023-05-03-cais-cluster-documentation.md:309]: Add new lines around code parts.

File: _posts/2023-05-03-cais-cluster-documentation.md [hunk 3]
```diff
@@ -300,6 +301,21 @@ gcc --version # if you print it out again here it'll be version 10
...
+#SBATCH --mail-user=mailto:user@example.com,slack:U12345ABCDE
+#SBATCH --mail-type=ALL
+```
+Feel free to replace the example email address and slack member id with your personal ones for the CAIS Compute Cluster workspace ([how to find your member id](https://www.workast.com/help/article/how-to-find-a-slack-user-id/)). You can also opt out of either notification service by removing it from the command as below.
```
Comment (steven-basart) [inline _posts/2023-05-03-cais-cluster-documentation.md:313]: Opt out? I'm confused by this. If they don't add that line they'll still get email and slack notifications?

**Review by steven-basart — CHANGES_REQUESTED** [review]

**Comment by steven-basart** [comment]
Not as important for documentation but in general please squash or "fixup" GitHub commits before merging.

File: _posts/2023-05-03-cais-cluster-documentation.md (hunk 1 above)
Comment (rumiahkessel) [reply _posts/2023-05-03-cais-cluster-documentation.md:30]: I had one installed that was being really annoying and reformatting all of those whenever I saved the file, so I uninstalled it and thought I undid all its changes. Will fix.

File: _posts/2023-05-03-cais-cluster-documentation.md (hunk 3 above)
Comment (rumiahkessel) [reply _posts/2023-05-03-cais-cluster-documentation.md:313]: I'll make it clearer. What I meant was they can choose to have email, slack, email and slack, or neither and I was explaining how they can setup only one or the other. I'll reword it to be clear.

**Review by steven-basart — COMMENTED** [review]
Ready to squash
Looks good to me (LGTM)

File: _posts/2023-05-03-cais-cluster-documentation.md (hunk 2 above)
Comment (rumiahkessel) [reply _posts/2023-05-03-cais-cluster-documentation.md:314]: added.

**Review by andriy-safe-ai — APPROVED** [review]
Provided some changes offline and helped with the squash. 

LGTM!

**Comment by steven-basart** [comment]
Oh no! Please don't merge it until the code is actually merged :(
